docs/byoh-instance-pre-requisites.md

```diff
@@ -4,6 +4,7 @@ The following pre-requisites must be fulfilled in order to add a Windows BYOH no
 * The Docker container runtime must be installed on the instance.
 * The instance must be on the same network as the Linux worker nodes in the cluster.
 * Port 22 must be open and running [an SSH server](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).
```

Comment:
> ... and running [an SSH server]

Given we are providing links to docs targeting OpenSSH, do we want to enforce OpenSSH as the SSH server? or could be any SSH server?